PWGHF/Tasks/taskX.cxx

```diff
@@ -108,6 +120,7 @@ struct TaskXMC {
 
   Configurable<int> d_selectionFlagX{"d_selectionFlagX", 1, "Selection Flag for X"};
   Configurable<double> cutYCandMax{"cutYCandMax", -1., "max. cand. rapidity"};
+  Configurable<bool> d_modeXToJpsiToMuMuPiPi{"d_modeXToJpsiToMuMuPiPi", false, "Perform Jpsi to mu+mu- analysis"};
```

Comment:
Please follow the official naming conventions.

Reply:
I will  move from d_modeXToJpsiToMuMuPiPi to modeXToJpsiToMuMuPiPi

Reply:
OK